typedoc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "entryPoints": [
+        "src/extension.ts",
+        "src/core/commandRunner/CommandRunner.ts",
+        "src/core/inferFullTargets.ts",
+        "src/core/updateSelections/RangeUpdater.ts",
+        "src/core/updateSelections/updateRangeInfos.ts",
+        "src/core/updateSelections/updateSelections.ts",
+        "src/processTargets/index.ts",
+        "src/processTargets/modifiers/processModifier.ts",
+        "src/processTargets/processMark.ts",
+        "src/typings/Types.ts",
+        "src/typings/treeSitter.ts",
+        "src/util/canonicalizeAndValidateCommand.ts",
+        "src/util/getPrimitiveTargets.ts"
+    ],
+    "readme": "none", 
```

Comment:
so we can set this so that we get a better landing page?

Reply:
Yes! That is the plan. Especially if we generate all apis it will be overwhelming without pointing to the key classes and functions.